app/views/products/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= render partial: 'shared/nav', locals: { h1_content: "Transactions" } %>
+<button class='cat_back'>
+  <%= link_to "Back", categories_path %>
+</button>
+
+<div class='pro_cat'>
+  <h2>
+    <%= @category_name%>  
+  </h2>
+  <p class='cat-amount'>
+   - Total: $ <%= @products.sum(:amount)%>
+  </p>
+</div>
```

Comment:
- [ ] Kindly position this div at the top and align the category name and the sum (total) properly matching the given design.

![your-total](https://github.com/oguntayomicheal/Budget-App/assets/53124548/d8f01a85-15c1-4916-8a45-49ad8a23209a)


![total-tr](https://github.com/oguntayomicheal/Budget-App/assets/53124548/45a0a69e-b936-42a0-a0bf-50bbfb06487b)